flexibleengine/provider.go

```diff
@@ -522,6 +530,9 @@ func Provider() *schema.Provider {
 			"flexibleengine_rds_account":            rds.ResourceMysqlAccount(),
 			"flexibleengine_rds_database":           rds.ResourceMysqlDatabase(),
 			"flexibleengine_rds_database_privilege": rds.ResourceMysqlDatabasePrivilege(),
+
+			"flexibleengine_sdrs_protection_group":   sdrs.ResourceProtectionGroup(),
+
```

Comment:
SDRS should create a new patch. Please remove these lines,

Reply:
Done， I has removed these lines